### ISSUE TITLE
Update README

### ISSUE THREAD

**Issue by Splaktar** [xlts-dev/verdaccio-google-cloud-storage#8]
- Update the readme file with some better docs on how to use this
- remove the experimental warnings
- Update/remove the badges
- update the example commands

**Comment by edclement** [comment]
@Splaktar I think all of those bullet points have been addressed in #6 with the exception of:
> update the example commands

We need to publish this to the public npm registry in order for those commands to be valid. I think that task should likely occur at minimum after we have working unit tests. I'm thinking we should just open a new issue for npm publishing and mark it as blocked by #3. We could then close this issue.
